Vitis_Platform_Creation/Design_Tutorials/03_Edge_VCK190/step1.md

```diff
@@ -15,27 +15,137 @@
 -->
 
 
-<table class="sphinxhide" width="100%">
```

Comment:
Please use the latest header of markdown files. It's required by documentation team to generate proper HTML.